BroadleafCommerce/BroadleafCommerce
# ContentItem StructuredContent required field validation

BroadleafCommerce/QA#1888
#### Tests
1. Make the `messageText` field in `BLC_FLD_DEF` required (via database change)
2. Navigate to a Content Item
3. Set the widget to `Text Widget`
4. Confirm that the `Message Text` field has the required field indicator
5. Save with the `Message Text` field empty 
6. Confirm that the required field validation hits & shows on the form
7. Navigate to a Widget
8. Hit `Add` on `Allowed Content Types`
9. Confirm that the ListGrid title reads `STRUCTURED CONTENT TYPE`

**Note**
The following issues were made to cover additional issues in this area:
https://trello.com/c/ZRUIMHVi and https://trello.com/c/OYePlkxp
